.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Build & Test Pop-CLI
+name: ci
```

Comment:
Why this change?

Reply:
Better naming for all the CI checks, which are slowly growing to be more than just "build and test". It also keeps the name short, so when viewed in the checks list below it is easier to parse.